sdk/core/azure-core/azure/core/pipeline/policies/_distributed_tracing.py

```diff
@@ -93,7 +90,9 @@ class DistributedTracingPolicy(SansIOHTTPPolicy[HTTPRequestType, HTTPResponseTyp
 
     # Azure attributes
     _REQUEST_ID = "x-ms-client-request-id"
```

Comment:
are these still needed then ?

Reply:
Yep, these are the keys in the actually HTTP headers. They are now just mapped to different attribute names on the spans.